### PR TITLE
Fixed Annotation component Layout shift

### PR DESCRIPTION
**Pull request by kwongz**
### Description
Original Bug: 
https://sf311.evidence.app/neighborhoods/OUTER%20SUNSET/

{114A2F83-3E59-4DD0-8D15-E9BAA8922CE7}

{E73CD15C-1406-41D3-BBB0-A0B01052E3FD}

Before:

Annotation deferring to default skeleton when loading in data, causing double loading skeletons to appear

{5FBE3589-D6D1-487E-A94D-1471064E6DC4}

{87CDD826-356F-453C-B562-D78B941EE35C}

{BDB5B5FE-69A2-4CC7-9C5C-CE153EF662F9}

{546B62F7-70F8-4C1F-8714-2FA1E4542818}

After:

Added custom skeleton in annotation components to remove this behavior

{7851A5BD-F437-4979-94DD-4D8245B24CEC}

{1D27547E-9B5B-49FD-9C32-02F2A19BE68C}

### Checklist

- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [ ] I have added to the docs where applicable
- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable

### CODE REVIEW

**Comment by kwongz** [comment]
{6D8EAD90-81C7-4AFC-97A4-E52830AF9DBD}

Still occuring with change

**Review by ItsMeBrianD — CHANGES_REQUESTED** [review]
Got a few stray console logs in here

**Comment by kwongz** [comment]
No More Layout shift

https://www.loom.com/share/533655bf94e6479482dddbeac346662d?sid=4a58dd53-393e-4b62-aed3-82945c738fe0

File: packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceArea/ReferenceArea.stories.svelte [hunk 1]
```diff
@@ -305,13 +298,79 @@
 		query
 	)}
 	<LineChart x="x" y="y" {data}>
-		<QueryLoad data={referenceAreaData}>
```
Comment (kwongz) [inline packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceArea/ReferenceArea.stories.svelte:308]: Query load already exists in annotation components, causing skeletons to render unnecessarily in stories

File: packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceArea/ReferenceArea.stories.svelte [hunk 2]
```diff
@@ -305,13 +298,79 @@
...
 				xMin="xMin"
-				xMax="non-existent-column"
+				xMax="xMax"
+				yMin="yMin"
+				yMax="yMax"
 				label="label"
 			/>
-		</QueryLoad>
-	</LineChart>
+		</LineChart>
```
Comment (kwongz) [inline packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceArea/ReferenceArea.stories.svelte:324]: Depicts similar env as bug experienced https://sf311.evidence.app/neighborhoods/OUTER%20SUNSET/

**Review by ItsMeBrianD — APPROVED** [review]

**Comment by hughess** [comment]
1 thing I think is worth checking - are the annotation components still reactive to changes in data? 

I think we had that problem before changing them to QueryLoad, so just want to make sure we aren't regressing that

**Comment by kwongz** [comment]
Annotations accepting dynamic Data from dropdown

image
image

**Comment by kwongz** [comment]
Hi @hughess, sorry for the miscommunication, Annotations accepting dynamic data is working alongside the layout shift fix.
If this looks good its ready to merge.

Thanks!

Dynamic data w/ Annotation + Dropdown 

image


https://www.loom.com/share/1a350c9ba66643bdaceefc4d91de94de?sid=cd2ec098-b4e1-482a-b7c8-522a65fde27d

**Review by hughess — APPROVED** [review]

**Pull request merged**